contrib/inventory/ec2.py

```diff
@@ -721,6 +728,51 @@ def get_elasticache_replication_groups_by_region(self, region):
         for replication_group in replication_groups:
             self.add_elasticache_replication_group(replication_group, region)
 
+    def get_autoscaling_instances_by_region(self, region):
+        ''' Makes an AWS API call to the list of Autoscale instances
+         in a particular region with tags of their Autoscale group
+         if the "propagate_at_launch" is set '''
```

Comment:
This seems a roundabout way of saying 'tags of the instances in the autoscaling group' (tags without  propagate_at_launch won't be set on the instances, but that seems an unnecessary aside)